analysis.go

```diff
@@ -2,10 +2,16 @@ package main
 
 import (
 	"encoding/json"
-	"os"
+	"io/ioutil"
 	"path/filepath"
 )
 
+var (
+	analysisErrorNotDiscovered         = 0
```

Comment:
How about we call it some positive name, instead of double negative?

analysisOk?